cedar-policy-core/src/entities/json/entities.rs

```diff
@@ -343,59 +339,6 @@ impl<'e, 's, S: Schema> EntityJsonParser<'e, 's, S> {
                     };
                     Ok((k.clone(), rexpr))
                 }
-                EntitySchemaInfo::Action(action) => {
-                    // We'll do schema-based parsing assuming optimistically that
-                    // the type in the JSON is the same as the type in the schema.
-                    // (As of this writing, the schema doesn't actually tell us
-                    // what type each action attribute is supposed to be)
```

Comment:
I _think_ this is a behavior change -- suppose you have an action attribute of type `ip` in the schema, then with this code you're allowed to pass just `"192.168.0.1"` for that attribute in the entity data (ie, just a string), whereas after this code is removed, you'll need the `{ __extn: { ... } }` form.

however, this is probably inconsequential, because most users are probably not passing actions in their entity data if they are also passing the schema, since the schema contains the action info and we already automatically pull the proper action info from the schema when constructing entities.